### ISSUE TITLE
PDFs sometimes garbled at first rendering

### ISSUE THREAD

**Issue by adamryczkowski** [RelaxedJS/ReLaXed#44]
```
$ cd ReLaXed-examples/examples/book
$ /opt/npm/bin/relaxed book.pug 
Watching book.pug and its directory tree.

Processing detected change in /book.pug...
... done in 1.90s
```
After that I get 29 page pdf without any text (but the pictures are present). The rabbit is on the second page (not on the first).

Executed on Ubuntu 16.04 64 bit. npm 5.6.0.

**Comment by Zulko** [comment]
Have you tried modifying the file again, just a second time ?

**Comment by adamryczkowski** [comment]
Now it works!... Can you explain me, why I needed to process the file two times? And how many times will always be enough?

**Comment by Zulko** [comment]
Can you rename the issue to "PDFs sometimes garbled at first rendering". That's a thing that I observe from time to time too (only at first rendering) and that we will have to tackle at some point.

**Comment by Zulko** [comment]
I believe this is fixed in v0.1.4. The trick was to load the page while waiting for ``loaded`` and ``domcontentloaded`` at the same time. Feel free to reopen if you experience this again in a version >0.1.4.
